app/uk/gov/hmrc/helptosavefrontend/controllers/RegisterController.scala

```diff
@@ -129,6 +128,10 @@ class RegisterController @Inject() (val helpToSaveService:     HelpToSaveService
     }
   }(redirectOnLoginURL = routes.RegisterController.getCreateAccountPage().url)
 
+  def checkYourDetails(): Action[AnyContent] = authorisedForHtsWithInfo { implicit request ⇒ implicit htsContext ⇒
```

Comment:
would we need `authorisedForHtsWithInfo` here? Would we just not grab the user info from session if the person is eligible?